apps/fishing-map/features/map/overlays/rulers/rulers.hooks.ts

```diff
@@ -82,6 +83,13 @@ const useRulers = () => {
     [dispatchQueryParams, resetMapControlValue, rulers, setRuleStart, value]
   )
 
+  const getRulersCursor = ({ features }: { features: DeckLayerPickingObject[] | undefined }) => {
```

Comment:
As we are passing the features as params, and this helper is not doing anything related with the hook itself, what about moving it to map-interation.utils with closer to [this](https://github.com/GlobalFishingWatch/frontend/blob/deck-migration/draw-layer-leftovers/apps/fishing-map/features/map/map-interaction.utils.ts#L9)